src/main/java/uk/gov/ons/ctp/response/collection/exercise/domain/SupplementaryDatasetEntity.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.ons.ctp.response.collection.exercise.domain;
+
+import java.util.UUID;
+import javax.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import net.sourceforge.cobertura.CoverageIgnore;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+import org.hibernate.annotations.Type;
+import uk.gov.ons.ctp.response.collection.exercise.message.dto.SupplementaryDatasetDTO;
+
+/** Domain model object. */
+@CoverageIgnore
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "supplementarydataset", schema = "collectionexercise")
+public class SupplementaryDatasetEntity {
+
+  @Id
+  @GenericGenerator(
+      name = "supplementarydatasetid_seq",
+      strategy = "org.hibernate.id.enhanced.SequenceStyleGenerator",
+      parameters = {
+        @Parameter(name = "sequence_name", value = "collectionexercise.supplementarydatasetidseq"),
+        @Parameter(name = "increment_size", value = "1")
+      })
+  @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "supplementarydatasetid_seq")
+  @Column(name = "id")
+  private Integer id;
```

Comment:
Does this table need an integer ID? The SupplementaryDatasetID should be unique too